apps/src/templates/export/expo/App.js.ejs

```diff
@@ -105,7 +105,7 @@ export default class App extends React.Component {
             originWhitelist={['*']}
             allowFileAccess
             source={{uri: indexUri}}
-            style={styles.webView}
+            style={Platform.OS === 'ios' ? styles.webViewIOS : styles.webViewAndroid}
```

Comment:
This is a separate fix for the fact that applab apps had a black background on Android only. The HTML page itself doesn't seem to have a color set. That will need to be resolved within applab (perhaps as part of our themes work?)